chemprop/v2/data/splitting.py

```diff
@@ -31,7 +32,7 @@ def split_data(
     sizes: tuple[float, float, float] = (0.8, 0.1, 0.1),
     seed: int = 0,
     num_folds: int = 1,
-) -> tuple[Sequence[MoleculeDatapoint], ...]:
+) -> tuple[list[int], ...] | tuple[list[list[int], ...], ...]:
```

Comment:
a return signature of `tuple[T, ...]` implies a tuple of unknown length containing items of type `T`. In the first return mode of this function, we should note that the tuple of is _known_ length (`== 3`) even if it's annoying to type out.

Reply:
Thanks for the comment! It makes sense. I leave the type hints out as suggested in the comment above and move them into the docstring.